### ISSUE TITLE
Unable to install @aragon/cli

### ISSUE THREAD

**Issue by TalkingAntColony** [aragon/aragon-cli#257]
On OSX 10.14, node 10.13.0 (managed via nvm 0.25.1)

Debug log:

https://gist.github.com/TalkingAntColony/cf18e69278a26df6524897b6744db8ce

**Comment by rubelux** [comment]
Same here, Npm can't find ```go-ipfs``:

```
npm ERR! path /Users/beauty13/Documents/projects/goodsted/goodToenAragon/node_modules/go-ipfs/bin/ipfs
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod
npm ERR! enoent ENOENT: no such file or directory, chmod '/Users/beauty13/Documents/projects/goodsted/goodToenAragon/node_modules/go-ipfs/bin/ipfs'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 
```

**Comment by maxxflyer** [comment]
same here
npm ERR! path /home/qbert/.nvm/versions/node/v10.13.0/lib/node_modules/@aragon/cli/node_modules/go-ipfs/bin/ipfs
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod
npm ERR! enoent ENOENT: no such file or directory, chmod '/home/qbert/.nvm/versions/node/v10.13.0/lib/node_modules/@aragon/cli/node_modules/go-ipfs/bin/ipfs'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

**Comment by izqui** [comment]
Taking a look.

**Comment by izqui** [comment]
Looks like the update of `go-ipfs` to 0.4.18 is broken. We will have to pin to 0.4.17 until they fix it.

https://github.com/ipfs/npm-go-ipfs/issues/17

**Comment by tommycox** [comment]
In the meantime, the CLI works if installed and linked from source as detailed here (https://hack.aragon.org/docs/cli-usage.html#install)

**Comment by izqui** [comment]
@TalkingAntColony @rubelux @maxxflyer Just published v5.2.0 which has a fix for this. Thanks for reporting and help debugging!

Also @tommycox thanks for the fix upstream!
